### ISSUE TITLE
Android do not open the respective dialogs for evaluation in production 

### ISSUE THREAD

**Issue by ggsant** [britannio/in_app_review#79]
I followed the documentation exactly and I can't use this library.

Android problem:
The requestReview method just doesn't work. I followed the documentation and put the app up for internal testing and was unable to open the play store for evaluation. It doesn't work in production either.

IOS issue:
The requestReview method works in debug with the emulator, but not in production.

Flutter version 3.3.9
Android version 11
Ios version 16.1

**Comment by britannio** [comment]
Hello, I check and respond to most if not all issues here and you're not the first to complain about this so apologies for this.

As for Android, could you try testing via internal app sharing (instead of internal testing) and let me know if this makes any difference?

As for iOS, if it works in development then it should work in production, subject to the limitations mentioned at https://developer.apple.com/documentation/storekit/skstorereviewcontroller/3566727-requestreview.
TL;DR: The popup will only appear once in production.

**Comment by ggsant** [comment]
What I found strange is that another person on my team just tested the internal test and the dialog appeared for him. For my cell phone and for other people on the team, it did not appear even the first time.

**Comment by ggsant** [comment]
I also added the permissions I saw in one of the open issues in proguard and it didn't work

**Comment by ggsant** [comment]
Hello, first I would like to apologize because I discovered that the error has nothing to do with the lib. After several tests, I found out why the dialog only appears on some android devices.

The error happened because the device was connected to the play store with an account that was not gmail. I'm going to open a pull request adding this in the documentation, I think it would solve a problem that I saw in other issues where a guy described the same situation.

I apologize for being harsh on the issue, it was a very difficult bug to find out what it was.
